cli/module_graph2.rs

```diff
@@ -603,6 +624,56 @@ impl Graph2 {
     Ok(())
   }
 
+  /// Given a string specifier and a referring module specifier, provide the
+  /// resulting module specifier and media type for the module that is part of
+  /// the graph.
+  pub fn resolve(
+    &self,
+    specifier: &str,
+    referrer: &ModuleSpecifier,
+  ) -> Result<ModuleSpecifier, AnyError> {
```

Comment:
Well this method has nothing to do with an import map, so I am not sure a comment here would make sense...

When the graph is built, the import map is resolved, in the builder.  The graph itself is never aware of it, because it is the builder that interacts with the specifier handler to insert modules in the graph, with dependencies full resolved to the other entries in the graph.  Once the graph is built, the import map is discarded.

So I am not sure a comment here means anything.

Reply:
Okay, thanks for explanation, that makes sense